builder/vmware/iso/builder.go

```diff
@@ -117,6 +117,13 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 		b.config.FloppyFiles = make([]string, 0)
 	}
 
+	for _, path := range b.config.FloppyFiles {
+		if _, err := os.Stat(path); err != nil {
+			errs = packer.MultiErrorAppend(errs,
+				fmt.Errorf("Bad Floppy disk '%s': %s", path, err))
```

Comment:
The working can be slightly improved to something like _Bad floppy disk file:..._.